backend/module/chaos/serverexperimentation/rtds/rtds.go

```diff
@@ -116,8 +115,8 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (module.Module, er
 		snapshotCacheV3:      gcpCacheV3,
 		cacheRefreshInterval: cacheRefreshInterval,
 		rtdsLayerName:        rtdsLayerName,
-		totalStreams:         rtdsScope.Gauge("totalStreams"),
-		totalResourcesServed: rtdsScope.Counter("totalResourcesServed"),
+		totalStreams:         scope.Gauge("totalStreams"),
```

Comment:
Do you think we should add prepend with `rtds` like `rtds.totalStreams`?

Reply:
yeah thats up to yall but the stats namespace will look like so after this PR merges
`clutch.module.chaos.experimentation.rtds.totalStreams` as an example.